easybuild/easyblocks/i/imkl.py

```diff
@@ -344,10 +348,10 @@ def sanity_check_step(self):
         if self.cfg['interfaces']:
             compsuff = '_intel'
             if get_software_root('icc') is None:
-                if get_software_root('PGI'):
-                    compsuff = '_pgi'
-                elif get_software_root('GCC'):
+                if get_software_root('GCC'):
```

Comment:
Same comment as before, I was wary of how GCCcore would impact here so the order was deliberate